packages/salesforcedx-apex-replay-debugger/src/core/logContext.ts

```diff
@@ -99,6 +113,98 @@ export class LogContext {
     );
   }
 
+  public getHeapDumps(): ApexHeapDump[] {
+    return this.apexHeapDumps;
+  }
+
+  public scanLogForHeapDumpLines(): boolean {
+    const heapDumpRegex = RegExp(/\|HEAP_DUMP\|/);
+    this.logLines.forEach((line, index) => {
+      if (heapDumpRegex.test(line)) {
+        const splitLine = line.split('|');
+        if (splitLine.length >= 7) {
+          const heapDump = new ApexHeapDump(
+            splitLine[3] /* heapDumpId */,
+            splitLine[4] /* className */,
+            splitLine[5] /* namespace */,
+            Number(splitLine[6]) /* line */
+          );
+          this.apexHeapDumps.push(heapDump);
+        } else {
+          // With the way log lines are, this would only happen
+          // if the user manually edited the log file.
+          this.session.printToDebugConsole(
+            nls.localize(`malformed_log_line', ${index + 1}, ${line}`)
+          );
+        }
+      }
+    });
+    return this.apexHeapDumps.length > 0;
+  }
+
+  public async fetchOverlayResultsForApexHeapDumps(
+    projectPath: string
+  ): Promise<boolean> {
+    let success = true;
+    try {
+      const orgInfo = await new ForceOrgDisplay().getOrgInfo(projectPath);
+      const requestService = new RequestService();
+      requestService.instanceUrl = orgInfo.instanceUrl;
+      requestService.accessToken = orgInfo.accessToken;
+
+      for (const heapDump of this.apexHeapDumps) {
+        this.session.printToDebugConsole(
+          nls.localize(`fetching_heap_dump', ${heapDump.toString()}`)
```

Comment:
There are two backticks and a single quote. How does this message look in the debug console?

Reply:
Ack! It's leftover from copy/paste. It's weird that the syntax checker didn't flag that. The same problem was on 137 which I've fixed as well. Good catch.